### PR TITLE
WHOOPS.  Emergency fix for character prefs.

### LINKED ISSUE #15319: Player Setup doesn't save
(WEB REPORT BY: ashemmes REMOTE: 172.93.109.202:7777)
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
When saving a character's changes, name, age, nanotrasen relation and handicaps do not save, and upon reload, return to the previous form.
Limbs, hair, underwear etc seem to save though.

>What you expected to happen
For character data to save.

>What actually happened
Only some of it does.

>Steps to reproduce if possible
Make changes, save, reload?

### PR DESCRIPTION
**Pull request by N3X15**
Fixes #15319

### CODE REVIEW

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -706,11 +706,11 @@ AND players.player_slot = ? ;"}, ckey, slot)
...
+	if(check.Execute(db))           // WHEN ADDING NEW COLUMNS, REMEMBER THE BLOCK AT THE END OF THE LINE --->
+		if(!check.NextRow())          //1         2           3         4          5               6       7    8        9         10          11          12          13                 14            15                   16                   1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16
+			q.Add("INSERT INTO players (player_ckey,player_slot,ooc_notes,real_name, random_name,    gender, age, species, language, med_record, sec_record, gen_record, player_alt_titles, disabilities, nanotrasen_relation, random_body) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)",
+                                  //1         2           3         4          5               6       7    8        9         10          11          12          13                 14            15                   16
+                                  ckey,       slot,       metadata, real_name, be_random_name, gender, age, species, language, med_record, sec_record, gen_record, altTitles,         disabilities, nanotrasen_relation, be_random_body)
```
Comment (N3X15) [inline code/modules/client/preferences_savefile.dm:713]: THIS LOOKS BETTER IN ATOM TRUST ME 😅

Comment (Exxion) [reply code/modules/client/preferences_savefile.dm:713]: >Atom

Comment (gbasood) [reply code/modules/client/preferences_savefile.dm:713]: It looks better in any editor that doesn't word wrap, yes

**Comment by Exxion** [comment]
Fixes #15319

**Comment by gbasood** [comment]
our SQL statements are officially a fucking mess

**Pull request merged**